datasette/templates/_codemirror_foot.html

```diff
@@ -1,38 +1,43 @@
 <script>
-window.onload = () => {
+  window.onload = () => {
```

Comment:
Should be a separate commit, but switching this to DOMContentLoaded should prevent the flash-of-uncodemirrored-object that can happen between the first render and the load event firing

Reply:
That sounds great to me.

Reply:
Filed https://github.com/simonw/datasette/issues/1894 to track